SwiftSyntaxDevUtils/Sources/swift-syntax-dev-utils/common/ScriptExectutionError.swift

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+struct ScriptExectutionError: Error {
```

Comment:
I would also make this convertible to `CustomStringConvertible`. It makes the errors show up a little nicer on the command line.